### PR TITLE
Merge release-20230427.061325 back into pr_bump_minor_0_2

### PR DESCRIPTION
**Pull request by holochain-release-automation2**
Please double-check the consistency of the CHANGELOG.md files

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,86 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # \[Unreleased\]
 
+# 20230427.061325
+
```
Comment (steveeJ) [inline CHANGELOG.md:10]: @abe-njama @ThetaSinner @jost-s 

this is a good place to give some general release notes for this minor release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,86 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+## [holochain\_cli-0.2.0](crates/holochain_cli/CHANGELOG.md#0.2.0)
+
+## [holochain\_cli\_signal\_srv-0.2.0](crates/holochain_cli_signal_srv/CHANGELOG.md#0.2.0)
+
+## [holochain\_cli\_sandbox-0.2.0](crates/holochain_cli_sandbox/CHANGELOG.md#0.2.0)
+
+## [holochain\_cli\_bundle-0.2.0](crates/holochain_cli_bundle/CHANGELOG.md#0.2.0)
+
+## [holochain-0.2.0](crates/holochain/CHANGELOG.md#0.2.0)
+
```
Comment (steveeJ) [inline CHANGELOG.md:20]: @abe-njama @ThetaSinner @jost-s 

alternatively here

File: CHANGELOG.md (hunk 1 above)
Comment (abe-njama) [reply CHANGELOG.md:10]: Second beta release.

File: CHANGELOG.md (hunk 2 above)
Comment (abe-njama) [reply CHANGELOG.md:20]: Or do we want to speak to what's included in this release - at a high level or at a detailed level.

File: CHANGELOG.md (hunk 1 above)
Comment (steveeJ) [reply CHANGELOG.md:10]: @abe-njama yes those two words can certainly be part of a paragraph :smile: 

more importantly we could communicate what that actually means in terms of

* key features introduced
* known limitations
* types of bugs/features that might be introduced during the 0.2 series
* what this means for the existing 0.1 series

Comment (abe-njama) [reply CHANGELOG.md:10]: A more secure version of Holochain that incorporates advanced network security protocols using WebRTC End-to-End Encryption and addresses known Holochain security vulnerabilities. This release introduces app-level blocking, for when an application wants to allow agents to block each other for reasons other than validation failures, user experience improvements that provide better network information, and improved system validation to make the network more secure and reliable.

@steveeJ

Comment (steveeJ) [reply CHANGELOG.md:10]: nice, thanks. i rephrased it a little, WDYT?

---

This release features  more secure version of Holochain that incorporates advanced network security protocols using WebRTC End-to-End Encryption and addresses known Holochain security vulnerabilities.

More specifically, it introduces introduces:
* app-level blocking, for when an application wants to allow agents to block each other for reasons other than validation failures
* user experience improvements that provide better network information
* and improved system validation to make the network more secure and reliable.

Comment (abe-njama) [reply CHANGELOG.md:10]: I like the rephrase. Looks better now.

Comment (steveeJ) [reply CHANGELOG.md:10]: cool, incorporated into #2290

**Pull request closed without merging**